azext_iot/_help.py

```diff
@@ -295,6 +295,30 @@
         --auth-type identity --identity {managed_identity_resource_id}
 """
 
+helps[
+    "iot hub device-identity generate-mqtt-credentials"
+] = """
+    type: command
+    short-summary: Generate user credentials for MQTT Device sign in.
+    long-summary: For more information, see
+                  https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-identity-registry#import-and-export-device-identities
```

Comment:
This link https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-identity-registry#import-and-export-device-identities is unrelated to mqtt connections.